lib/sequent/core/helpers/attribute_support.rb

```diff
@@ -63,12 +63,11 @@ def attrs(args)
                 Sequent::Core::Helpers::DefaultValidators.for(type).add_validations_for(self, attribute)
               end
 
-              if type.instance_of?(Sequent::Core::Helpers::ArrayWithType)
-                associations << attribute
-              elsif included_modules.include?(ActiveModel::Validations) &&
-                    type.included_modules.include?(Sequent::Core::Helpers::AttributeSupport)
-                associations << attribute
-              end
+              next unless type.instance_of?(Sequent::Core::Helpers::ArrayWithType) ||
+                          (included_modules.include?(ActiveModel::Validations) &&
+                              type.included_modules.include?(Sequent::Core::Helpers::AttributeSupport))
+
+              associations << attribute
```

Comment:
Not sure if this is better, `unless` already breaks my brain and then we have this complicated condition...

Reply:
I agree, it was rubocop's suggestion...